xml.go

```diff
@@ -32,7 +32,12 @@ type Run struct {
 	TaskProgress     []TaskProgress `xml:"taskprogress" json:"task_progress"`
 	TaskEnd          []Task         `xml:"taskend" json:"task_end"`
 
-	rawXML []byte
+	NmapErrors []NmapErrors
+	rawXML     []byte
+}
+
+type NmapErrors struct {
```

Comment:
This structure name should be singular and not plural, but a slice of those would be plural

```suggestion
type NmapError struct {
```